src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -279,7 +279,8 @@ public void validateAppEngineJavaComponents()
       throws AppEngineJavaComponentsNotInstalledException {
     if (!Files.isDirectory(getJavaAppEngineSdkPath())) {
       throw new AppEngineJavaComponentsNotInstalledException(
-          "Validation Error: Java App Engine components not installed");
+          "Validation Error: Java App Engine components not installed."
+              + " Fix by running 'gcloud components install app-engine-java' on command-line.");
```

Comment:
I would revisit this when these exceptions are made checked. For the time being, however, since the clients are not forced to handle the exception, I'd rather have this in there to potentially save time for users encountering this issue.